### DISCUSSION TITLE
Ok...

### DISCUSSION THREAD

**Discussion by Yvonnecb** [community/community#83924] [Other Feature Feedback, Questions, & Ideas] [answered]
I would like to know why my information goes back at least 11 years ago?!

**Comment by Shahnawazk7944** [comment]
**It is possible that your information goes back at least 11 years ago on GitHub because that is when you first created your account or started using the platform. GitHub keeps a record of all your activities, including code contributions, repositories, and other interactions, so your information may go back as far as when you first joined the platform. If you have specific concerns about the information on your GitHub account, you can reach out to their support team for further assistance.**

**Reply by Yvonnecb** [reply]
Yes, it's completely possible that my data goes back at least 11 years. I can say this because I have seen code going back that far in my account with them. I only recently reached out to GitHub because I need security and I don't know exactly what they do. But I remember that a few years ago I caught them broadcasting me Globally and I didn't appreciate it at all. I didn't know what to do to make it stop so I did a pull request! I'm sure it was a huge joke on their part. 
Not so funny to me though. 
I also discovered that my roommate was working with them.
GitHub has been involved for too long. They have been doing work with Open Source License and I have found more licenses in my phone than anyone I know. 

I don't understand why they have been obsessed with me but I have been acting like a bitch in the hopes that they leave me alone and I would like to be compensated for my part, albeit unknown, for my contributions over the past 11 years from the corporations that have profited from the use of my images, videos, audible contributions to all of these companies. I just can't help but wonder what I've been worth to them?

**Reply by Shahnawazk7944** [reply, marked as answer]
I don't have words for you, There's a famous quote `If you don't pay for the product, you are the product` you are a great example of this😁

**Reply by Yvonnecb** [reply]
So I'm a product is what you're saying. That's not a good feeling to discover that you've been the "product" without my Gmail or knowledge. I never received any form of compensation for my contributions. I think it's perfectly okay for me to get some.

**Reply by Shahnawazk7944** [reply]
Ohh😅 bro, don't take my words seriously. It's just my point of view. There might be a bug or another reason causing this. You should contact GitHub support; they'll definitely help you. are you RAW agent? or a hacker?😁

**Comment by Yvonnecb** [comment]
I have been the "product" for 11 flipping years! I didn't know that.
Are you saying if you found out tomorrow that you have been stalked for the
past 11 years you wouldn't bash an eyelash?

Yvonne C Berger

**Comment by Boneheinwai6801** [comment]
because of your appearances congregation

**Comment by Yvonnecb** [comment]
On Wednesday, January 10, 2024, Shahnawaz khan ***@***.***>
wrote:

> Ohh😅 bro, don't take my words seriously. It's just my point of view.
> There might be a bug or another reason causing this. You should contact
> GitHub support; they'll definitely help you. are you RAW agent? or a
> hacker?😁
>
> —What is RAW agent? I'm definitely not a hacker! I am a woman who has just
> discovered that I have been being broadcast and used as a model for
> Advertising and Marketing Campaigns Globally for the past 11 years now. So
> I'm just a little confused. I haven't been paid for it yet but I hope they
> will be willing to pay me now.
>
I've actually made quite a bit of money for these companies because they
obviously still want to use me. I don't know why? I'm 55 years old now.
When this started, as far as I know, I was 44 years old and I looked a lot
better. I just wanted to start my own company doing digital work and
modeling for advertising campaigns and marketing campaigns. I didn't know
all this other stuff was happening. I just thought I had A stalker. I
didn't realize it was a whole team against me. I'm only using my Android
11. Lol I don't know the first thing about coding and it's not anything I'm
passionate about to be honest with you.
Well, I'm sorry if I can't respond right away but like I said someone else
has been in my settings and I don't know how to change them back so I'm
getting all of my notifications and texts.H
Have a great day wherever you are!

**Reply by Shahnawazk7944** [reply]
_I'm really sorry for taking all this in fun way, you should take some legal actions._

**Reply by Yvonnecb** [reply]
Yes, I have been hoping to avoid that way and perhaps continue to work with GitHub and PayPal, Google and Meta. I don't want to be in the spotlight in my bathroom or while I'm in the nude in the privacy of my bedroom anymore then anyone else would. 
I am learning so much about the way I've been being used for brand marketing campaigns and advertising campaigns and projects internationally. 
It's been a real trip?
I'm not really interested in learning how to write codes but I'm thinking that I don't need to since I've never had to in my past but I'm still figuring out what made me so special??? I don't know if you've seen me lately but I'm a complete mess! I don't know who would want to continue to use me for Advertising or anything but I just wanted to make sure I'm secure with my future assets!. I don't own a house, my car hasn't ran in years and I'm going to be needing some retinol insurance so I know I'm taken care of for my future assets and I would like my children to be taken care of also. 
After all the work I've been involved with I honestly don't think that's too much to ask. 
That's just what I would like. 

But you're right, I could always turn to a law firm that specializes in international issues such as mine. 

Thanks for your support! 
I'm glad someone else is on my side and I appreciate it! 

Yvonnecb

**Reply by Ycberger** [reply]
Someone very kind said, "https://github.com/orgs/community/discussions/83924?sort=old#discussioncomment-7996457"
